### ISSUE TITLE
run_RF2NA.sh: Logging issues

### ISSUE THREAD

**Issue by SuhasSrinivasan** [uw-ipd/RoseTTAFold2NA#106]
https://github.com/uw-ipd/RoseTTAFold2NA/blob/0e3742355cd48ff1823e2b6a3893b5a07598d16d/run_RF2NA.sh#L116-L118

There are two issues in the snippet:
1. The `echo` for `make_rna_msa.sh` has been copied over
2. There are no stdout and stderr logs for `make_pMSAs_prot_RNA.py`

Suggested change
```
    echo "Creating joint Protein-RNA MSA"
    echo " -> Running command: $PIPEDIR/input_prep/make_pMSAs_prot_RNA.py $WDIR/$lastP.msa0.a3m $WDIR/$lastR.afa $WDIR/$lastP.$lastR.a3m"
    $PIPEDIR/input_prep/make_pMSAs_prot_RNA.py $WDIR/$lastP.msa0.a3m $WDIR/$lastR.afa $WDIR/$lastP.$lastR.a3m > $WDIR/log/make_pMSA.$tag.stdout 2> $WDIR/log/make_pMSA.$tag.stderr
```

**Comment by SuhasSrinivasan** [comment]
Submitted a Pull Request #111

**Comment by SuhasSrinivasan** [comment]
Fixed in PR #111
